lms/djangoapps/grades/management/commands/ai_grading_hack.py

```diff
@@ -0,0 +1,29 @@
+"""
+A hackathon-specific management command. I've found some nifty test data on
+Kaggle (see https://www.kaggle.com/c/asap-aes) to use for training our AI
+grader; this command will insert that data as a bunch of staff-graded assessments.
+
+This command (in edx-platform) will create all the users, and enroll them in the course.
+
+The remainder is handled by an ora2 command.
+"""
+
+from django.core.management.base import BaseCommand
+
+from enrollment.api import add_enrollment
+from student.tests.factories import UserFactory
+
+
+class Command(BaseCommand):
```

Comment:
for some reason, I can't run any cms management commands. This doesn't actually use grades at all, I just needed an lms location in which to run this hack.